app/packages/looker/src/elements/common/options.ts

```diff
@@ -58,11 +58,13 @@ export class OptionsPanelElement<
   }
 }
 
-export class LoopVideoOptionElement extends BaseElement<VideoState> {
+export class LoopVideoOptionElement<
+  State extends ImaVidState | VideoState = VideoState
```

Comment:
I assume this is needed to make TS happy somewhere? Would probably be better to declare the required interface since these types only partially overlap, but not a blocker.

Reply:
Yeah this is tech debt that should be re-evaluated in future work